backend/api/rpc.go

```diff
@@ -0,0 +1,66 @@
+package api
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"net/rpc"
+	"os"
+	"time"
+)
+
+type RPCEndpoints int
+
+type Signal struct {
+	Status string
+	Msg    string
+}
+
+// initRPCServer initialized the RPC server that listens to incoming LCM tasks
+// by the RPC clients
+func InitRPCServer() error {
+	endpoints := new(RPCEndpoints)
+	// Publish the receivers methods
+	err := rpc.Register(endpoints)
+	if err != nil {
+		return fmt.Errorf("Format of service Task isn't correct. Reason: %v", err)
```

Comment:
```suggestion
		return fmt.Errorf("format of service Task isn't correct. Reason: %v", err)
```

https://github.com/golang/go/wiki/CodeReviewComments#error-strings
> That is, use `fmt.Errorf("something bad")` not `fmt.Errorf("Something bad")`, so that `log.Printf("Reading %s: %v", filename, err)` formats without a spurious capital letter mid-message. This does not apply to logging, which is implicitly line-oriented and not combined inside other messages.